modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -561,4 +566,30 @@ public String escapeQuotationMark(String input) {
     public String escapeUnsafeCharacters(String input) {
         return input.replace("*/", "*_/").replace("/*", "/_*");
     }
+
+    @Override
+     public void postProcessFile(File file, String fileType) {
+         if (file == null) {
```

Comment:
i suggest to call the parent method, too, since there could be some logic added later.

Reply:
Sure. I'll do it later.

Reply:
Done via 9573d8e